loleaflet/src/control/Control.Menubar.js

```diff
@@ -1522,6 +1522,8 @@ L.Control.Menubar = L.Control.extend({
 		} else if (id === 'shareas') {
 			this._map.openShare();
 		} else if (id === 'print') {
+			// print
+			// Whether the print handler is active (for Chrome).
```

Comment:
This seems to be the wrong place to add those since we are just checking the id and then calling `this._map.print();` we need to find where that is declared in the code. 


Option | Type | Default | Description
-- | -- | -- | --
print | Boolean | true | Whether the print handler is active (for Chrome).

notice that we are talking about a boolean option

Reply:
Let me know where it is declared first